tests/testthat/test-packrat.R

```diff
@@ -279,12 +279,16 @@ withTestContext({
 
     # validate the installed package has properly annotated DESCRIPTION
     descpath <- file.path(libDir(projRoot), "falsy/DESCRIPTION")
-    desc <- as.data.frame(readDcf(descpath), stringsAsFactors = FALSE)
+    desc <- as.data.frame(readDcf(descpath))
```

Comment:
Is the removal of `stringsAsFactors = FALSE` here just to ensure that we're resilient against both strings + factor fields read from the DESCRIPTION?

Reply:
Yes. I could make these an explicit `stringsAsFactors = TRUE` if that's better.

Reply:
It is (this is clear from the PR description.)